.github/workflows/build.yml

```diff
@@ -78,8 +78,8 @@ jobs:
       - name: Build + Test
         if: github.ref != 'refs/heads/master'
         run: |
-          mvn -B -e -DXss4m -DXX:MaxRAMPercentage=90.0 -DskipTests=true install
-          mvn -B -e -DXss4m -DXX:MaxRAMPercentage=10.0 surefire:test -DargLine="-XX:MaxRAMPercentage=70.0"
+          mvn -B -e -DXss4m -DXX:MaxRAMPercentage=95.0 -DskipTests=true install
+          mvn -B -e -DXss4m -DXX:MaxRAMPercentage=10.0 surefire:test -DargLine="-XX:MaxRAMPercentage=80.0"
```

Comment:
We should be careful here, I and @kevin-m-knight-gs once pushed these to these numbers and I got intermittent test failure due to OOM (even on the large machine)